packages/core/src/discovery.js

```diff
@@ -95,6 +98,45 @@ function parseDomResources({ url, domSnapshot }) {
   }, new Map([[rootResource.url, rootResource]]));
 }
 
+async function parseImageSrcset(snapshot, discovery) {
+  let log = logger('core:snapshot');
+  const promiseList = [];
+  const { resources, meta, domSnapshot } = snapshot;
+  const { allowedHostnames } = discovery;
+  let { imageLinks } = domSnapshot;
+
+  imageLinks = imageLinks.filter((link) => hostnameMatches(allowedHostnames, link));
+  imageLinks = imageLinks.filter((link) => !resources.get(link));
+
+  log.debug(`Capturing image src set: ${JSON.stringify(imageLinks)}`, meta);
+  for (const link of imageLinks) {
+    const promise = makeRequest(link, { buffer: true, retries: 0 }, (body, response) => {
```

Comment:
This may or may not work depending on the Auth requirements of the backend. I would recommend doing this in discovery browser as that would auto take care of auth requirements, else we will have to replicate this. 

Also `request` by default uses proxy if set, which may or may not be required for browser [ in fact could have issues if requests go via proxy ]

Reply:
Actually calling this urls using fetch api inside browser is causing CORS issue, that's y written logic in such a way